pkg/rtc/transport.go

```diff
@@ -405,6 +417,24 @@ func (t *PCTransport) setICEConnectedAt(at time.Time) {
 		// This prevents reset of connected at time if ICE goes `Connected` -> `Disconnected` -> `Connected`.
 		//
 		t.iceConnectedAt = at
+
```

Comment:
We set `iceConnectedAt` only once as we wanted to track short ICE connection failure by measuring time since the first time a participant connected. Do we want to keep that? Or should we reset things at ICE restart and measure every time given the set of changes in this PR?

Reply:
Currently the udp/dtls issues reported are blocked all the time, so we can detect it at the first time. And we don't have `reason` for icerestart now to identify a signal disconnect or pc failure(need to reset the timer), so I left it here and we can observe the client reports about connectivity.